src/test/unit/controls/Graph/GraphGenerate-spec.js

```diff
@@ -165,7 +165,7 @@ describe("Graph - Generate", () => {
             };
             graph = new Graph(graphConfig);
             const contentContainer = d3.select(`.${styles.contentContainer}`);
-            expect(toNumber(contentContainer.attr("x"), 10)).toBeCloserTo(0);
+            expect(toNumber(contentContainer.attr("x"), 10)).toBeCloserTo(-16);
```

Comment:
17?

Reply:
It is because of `config.axisLabelWidths.y`.